app/services/l10n.js

```diff
@@ -1,19 +1,54 @@
 import computed from 'ember-computed';
 import L10n from 'ember-l10n/services/l10n';
-// import l10nFingerprintMap from './../utils/l10n-fingerprint-map';
+import l10nFingerprintMap from './../utils/l10n-fingerprint-map';
 
 export default L10n.extend({
 
   availableLocales: computed(function() {
     return {
-      'en': this.t('en')
+      'bn'      : this.t('Bengali/Bangla'),
+      'zh_Hans' : this.t('Chinese (Simplified)'),
+      'zh_Hant' : this.t('Chinese (Traditional)'),
+      'en'      : this.t('English'),
+      'fr'      : this.t('French'),
+      'de'      : this.t('German'),
+      'id'      : this.t('Indonesian'),
+      'ko'      : this.t('Korean'),
+      'pl'      : this.t('Polish'),
+      'es'      : this.t('Spanish'),
+      'th'      : this.t('Thai'),
+      'vi'      : this.t('Vietnamese'),
+      'hi'      : this.t('Hindi'),
+      'ja'      : this.t('Japanese'),
+      'ru'      : this.t('Russian')
     };
   }),
 
-  autoInitialize: true,
+  localStorageKey: 'current_locale',
+
+  autoInitialize: false,
 
   jsonPath: '/assets/locales',
```

Comment:
@geekyd I am aware there is an error, however it is not the same error as this one, it changed and hence I thought it might be worth discussing
![image](https://user-images.githubusercontent.com/17252805/27357882-a8ec8e16-5631-11e7-8617-3ce98d56b5c3.png)

This is the other error, only appears on localhost, not heroku deployment